hooks/useProfile.ts

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
```

Comment:
is this file carried over from the 1st sprint? if so, we may have to remove this file. (see above comment about removing useProfile from the `app/onboarding/page.tsx` as well)